resources/home/dnanexus/annotate_workbooks/annotate_workbooks_with_QC.py

```
@@ -159,11 +215,145 @@ def create_combined_qc(multiqc_path):
+
+    gene_depths, gene_pos = get_min_depth_per_gene(intersect_file)
+    if not gene_depths:
+        print(f"[WARN] No genes found for {intersect_file.name}")
+
+    try:
+        sample_workbook = openpyxl.load_workbook(workbook_path)
+        worksheet = sample_workbook["summary"]
+
+        for gene, depth in gene_depths.items():
+            pos = gene_pos[gene]
+            depth_result, pos_result = write_gene_depth_to_cell(
```

Ruff (0.15.20)

[warning] Unpacked variable `pos_result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)